.devcontainer/build.sh

```diff
@@ -74,4 +76,12 @@ CmakeBuild() {
   fi
 }
 
-[[ $SOURCED == "false" ]] && main "$@" || echo "Sourced!"
\ No newline at end of file
+if [[ $SOURCED == "false" ]]; then
+  # It is important to return exit code of main
+  # To be future-proof, this is handled explicitely
+  main "$@"
+  BUILD_RESULT=$?
+  exit BUILD_RESULT
```

Comment:
I think we need to access the variable explicitly
```suggestion
  exit $BUILD_RESULT
```